news/2025/2025-09-20-osumania-4k-world-cup-2025-semifinals-recap.md

```diff
@@ -0,0 +1,231 @@
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/artists/383/Songs/Kou_-_Ankh_Natron_Collider.mp3">
+    </audio>
+</div>
+
+Last, but certainly not least, we were given yet another tiebreaker! [Kou! - Ankh Natron Collider](https://osu.ppy.sh/beatmapsets/2432016#mania/5298404) by ::{ flag=US }:: [elexire](https://osu.ppy.sh/users/9206093), ::{ flag=CN }:: [V1do-](https://osu.ppy.sh/users/17527968), ::{ flag=US }:: [-mint-](https://osu.ppy.sh/users/8976576), ::{ flag=US }:: [Toaph Daddy](https://osu.ppy.sh/users/7616811), and ::{ flag=KR }:: [11Bit](https://osu.ppy.sh/users/14804526) brought quite the... storm? Or should we say dust storm? Either way, this tiebreaker is quite the opposite to [Kou!](https://osu.ppy.sh/beatmaps/artists/383)'s [last original tiebreaker](https://osu.ppy.sh/beatmapsets/2202823#fruits/4662299), with this year's being just as impactful. With Kou!'s very familiar song structure, this tiebreaker was without question able to bring out a ton of tough patterns, whether that be rice or long notes. Especially with the huge team behind it, with ::{ flag=IE }:: [naraicat](https://osu.ppy.sh/users/6419788) working on the illustrations, ::{ flag=CN }:: [AlexDunk](https://osu.ppy.sh/users/9194799) and ::{ flag=SG }:: [TheFunk](https://osu.ppy.sh/users/13981991) working on the graphic design, ::{ flag=NL }:: [Promachos](https://osu.ppy.sh/users/14069486) working on the storyboard, and ::{ flag=US }:: [Secondoverthree](https://osu.ppy.sh/users/13432062), ::{ flag=ID }:: [LenLitchu](https://osu.ppy.sh/users/34098325), and ::{ flag=CN }:: [Sakura006](https://osu.ppy.sh/users/10365024) all working on the video, this chart, and *especially* the ending are more than guaranteed to give just as much of a shock as they'll give you a challenge.
+
+## Semifinals summary
+
+| Team A |  |  | Team B |
+| --: | :-: | :-: | :-- |
+| Vietnam ::{ flag=VN }:: | 1 | **6** | ::{ flag=ID }:: **Indonesia** |
+| **South Korea** ::{ flag=KR }:: | **6** | 1 | ::{ flag=PE }:: Peru |
+| **Hong Kong** ::{ flag=HK }:: | **6** | 0 | ::{ flag=PH }:: Philippines |
+| Canada ::{ flag=CA }:: | 3 | **6** | ::{ flag=TH }:: **Thailand** |
+| **United States** ::{ flag=US }:: | **6** | 0 | ::{ flag=CL }:: Chile |
+| **South Korea** ::{ flag=KR }:: | **6** | 2 | ::{ flag=ID }:: **Indonesia** |
+| Hong Kong ::{ flag=HK }:: | 2 | **6** | ::{ flag=TH }:: **Thailand** |
+| **China** ::{ flag=CN }:: | **6** | 1 | ::{ flag=GB }:: United Kingdom |
+
+It's easy to see from the table above that only the best of the best teams are still standing. At this point in the tournament, matches are less likely to be decided by sweeping differences. And even if we've already seen quite a few close matches on prior rounds, this is soon going to be changing starting from this round onwards. Look no further than this week's [score highlights](#score-highlights)!
```

Comment:
if this is saying that sweeps are less likely, then it seems to contradict the table above that's mostly filled with sweeps. if the second sentence is about close matches being prevalent before but *not* from here on, then that contradicts the first sentence. i recommend reducing this to the main point, after figuring out what it is <.<

Reply:
smth about "while the results here seem like sweeps, many of the points in these matches were decided by very close score margins, which is something we've seen more in these later tournament rounds"

Reply:
```suggestion
While the results here seem like sweeps, many of the points in these matches were decided by incredibly close score margins, which is something we've been seeing more and more in these later tournament rounds. To appreciate those individual score differences, we've [compiled the highlights below](#score-highlights)!
```